package.json

```diff
@@ -10,18 +10,22 @@
     "axios": "^0.18.0",
     "blueimp-canvas-to-blob": "^3.14.0",
     "bootstrap": "^4.1.3",
+    "change-case": "^3.1.0",
     "emotion": "^9.2.10",
     "emotion-theming": "^9.2.9",
     "js-cookie": "^2.2.0",
     "lodash": "^4.17.11",
     "moment": "^2.24.0",
+    "pigeon-maps": "^0.13.0",
```

Comment:
Curious to hear what you thought about using pigeon maps...

Reply:
Pigeon maps was super easy to use - just plugged in the center coordinates into a Map component and a nested Marker component for the pin. The hardest part was making it responsive but there was a GH issue that spelled it out. This is a great library for quickly rendering simple maps.